bin/configurer/utils/idam-lease-user-token.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ "${IDAM_VERSION:-tactical}" == 'tactical' ]]; then
```

Comment:
It is bit of technical debt. In one of next steps we will stop using tactical IDAM and that will be the time when we remove this tech debt.